src/Workspaces/Core/Desktop/project.json

```diff
@@ -2,7 +2,9 @@
   "dependencies": {
     "ManagedEsent": "1.9.4",
     "Microsoft.Build": "15.1.0-preview-000458-02",
-    "Microsoft.Build.Tasks.Core": "15.1.0-preview-000458-02"
+    "Microsoft.Build.Tasks.Core": "15.1.0-preview-000458-02",
+    "System.Data.Common": "4.3.0",
+    "System.Data.SQLite": "1.0.104"
```

Comment:
do we need to make sure we are using same SQLite version as other team like service hub (setting host something?) - @AArnott since I am not sure how version conflict?